transient-aws/preloaded-ami-builder/build-ami.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/sh
```

Comment:
Govind and I had to make a few modifications to get these scripts to work and I wasn't sure about modifying the parent versions.   Let's sync up offline about this.